### PR TITLE
Show datetimes as humanized localtime ISO

### LINKED ISSUE #137: Use consistent DateTimes
The datepicker for setting an ack's expiry date is hardcoded to US dates.
It is hard-coded to MM/dd/yyyy. It should use whatever the locale uses. (Preferably by something looking up the locale's date-format and storing it in a variable so that it can be overridden by the members of the Church of ISO 8601 later =) )

### PR DESCRIPTION
**Pull request by hmpf**
Closes #137

### CODE REVIEW

File: src/components/incidenttable/FilteredIncidentTable.tsx [hunk 1]
```diff
@@ -235,7 +237,7 @@ const FilteredIncidentTable: React.FC<FilteredIncidentsTablePropsType> = ({
         paginationComponent={paginationComponent}
       />
       <p>
-        Last refreshed {lastRefresh === undefined ? "never" : lastRefresh.toLocaleTimeString()}, {autoUpdateText}
+        Last refreshed {lastRefresh === undefined ? "never" : formatTimestamp(lastRefresh).slice(11)}, {autoUpdateText}
```
Comment (jorgenbele) [inline src/components/incidenttable/FilteredIncidentTable.tsx:240]: Why do you use slice(11) here?

**Comment by jorgenbele** [comment]
Before you submit a PR you should run `yarn format` or equivalent using npm to reformat the code.

File: src/components/incidenttable/FilteredIncidentTable.tsx (hunk 1 above)
Comment (hmpf) [reply src/components/incidenttable/FilteredIncidentTable.tsx:240]: I'm not using the entire format. Microseconds are unwanted,as is timezone.

**Comment by hmpf** [comment]
Closed by #240

**Pull request closed without merging**